cms/src/components/content/ContentOverview.vue

```diff
@@ -32,13 +29,28 @@ const tagType = Object.entries(TagType).some((t) => t[1] == props.tagType)
     ? props.tagType
     : undefined;
 
-const contentParents = db.whereTypeAsRef<PostDto[] | TagDto[]>(props.docType, [], tagType);
 const languages = db.whereTypeAsRef<LanguageDto[]>(DocType.Language, []);
 const selectedLanguage = ref<Uuid>("");
 const languageOptions = computed(() =>
     languages.value.map((l) => ({ value: l._id, label: l.name })),
 );
 
+const queryOptions = ref<ContentOverviewQueryOptions>({
+    languageId: "",
+    parentType: props.docType,
+    tagType: tagType,
+    translationStatus: "all",
+    orderBy: "updatedTimeUtc",
+    orderDirection: "desc",
+    pageSize: 20,
+    pageIndex: 0,
+    tags: [],
+    search: "",
+    publishStatus: "all",
+});
+
+const queryKey = computed(() => JSON.stringify(queryOptions.value));
```

Comment:
Can't we move this to the `ContentTable` as well? It seems that the only reason why the query options are declared here is that we have the language selector in this component, and now it feels divided between this component and the table. We could pass the language along as a prop instead. (in any case I think the language selection should happen in the user profile, not here on the page).

Reply:
My thought was that the query and filtering controls can be added to `ContentOverview.vue`, and not to `ContentTable.vue`. That keeps `ContentTable` a bit cleaner, and sorting / filtering is applied through the `queryOptions`.

But this is debateable, so welcome to challenge my thinking (again).

Reply:
Hmm still a bit torn on this one. Intuitively I feel like filtering and sorting is an integral part of the table, and should thus be part of that component too (that is how Polaris does it as well). But if we can foresee that we need that info outside of the table then this approach makes sense. The way it currently is it just feels like it can all be packed into one component.

Reply:
I am a bit afraid creating one "super component", but perhaps we can create a separate filter component and include it in the table component to keep it a bit separated.

Reply:
Tried to move it, but it makes refreshing after changing query variables difficult, so reverted back to having the query options in `ContentOverview.vue`